sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -17,7 +17,7 @@ export interface GetSchemaPropertiesOptions extends OperationOptions {
 }
 
 // @public
-export const enum KnownSerializationType {
+export const enum KnownSchemaFormat {
```

Comment:
Why do we use "Known" here?

Reply:
It is a standard pattern in JS where we model extensible enums as strings and in their docstring we refer to a known enum that defines the possible values to use. This known enum does not show up anywhere else in the public surface.